packages/application-generic/src/usecases/get-novu-integration/get-novu-integration.command.ts

```diff
@@ -7,4 +7,8 @@ export class GetNovuIntegrationCommand extends EnvironmentWithUserCommand {
   @IsOptional()
   @IsEnum(ChannelTypeEnum)
   channelType?: ChannelTypeEnum;
+
+  @IsOptional()
+  @IsBoolean()
+  ignoreActiveCount? = false;
```

Comment:
```suggestion
  ignoreActiveCount = false;
```
Not sure if this can be done here but if setting the default value you don't need to add the question mark.

Reply:
It needs to be there otherwise typescript thinks it is always needed to be provided.

Reply:
I am so disappointed it behaves differently as the arguments in a function. 😞